jest-puppeteer.config.js

```diff
@@ -18,6 +18,12 @@ module.exports = {
   launch: {
     headless,
     slowMo: process.env.PUPPETEER_SLOWMO, // slow down tests with slowMo ms
+    args: [
```

Comment:
are you sure we need them? the e2e tests seem to run perfectly without these

Reply:
These weren't needed - just some trial & error based on some thread somewhere.